pkgs/top-level/python-packages.nix

```diff
@@ -11600,6 +11600,16 @@ self: super: with self; {
 
   pytest-asyncio = callPackage ../development/python-modules/pytest-asyncio { };
 
+  pytest-asyncio_0_21 = pytest-asyncio.overridePythonAttrs (old: rec {
```

Comment:
Do we expect this to be used by more than autobahn? if not we might as well make it local for autobahn since it is only used in checkInputs

Reply:
We might have to. The upstream issue makes it sound like autobahn isn't the only reverse dependency affected.